.github/workflows/ci.yaml

```diff
@@ -23,3 +23,5 @@ jobs:
           pip install -r gw_spaceheat/requirements/test.txt
       - name: Run tests with pytest
         run: pytest
+        env:
+          PYTHONPATH: gw_spaceheat:$PYTHONPATH
```

Comment:
Nice. We could use one of the online brokers for the github tests, so if we figure out how to copy files over in git actions then maybe ci will start to work.

Reply:
There's probably an open source recipe to start an mqtt broker, probably mosquitto in a docker container. I just need to do a little GitHub Actions reading before pushing that further.